lsp-scala.el

```diff
@@ -18,15 +18,20 @@
 (require 'sbt-mode)
 
 ;;;###autoload
-(defcustom lsp-scala-server-command '("metals-emacs")
-  "The command to launch the language server,"
+(defcustom lsp-scala-server-command "metals-emacs"
+  "The command to launch the Scala language server."
   :group 'lsp-scala
-  :type 'list)
+  :type 'file)
 
-(defcustom lsp-scala-workspace-root default-directory
-  "The root directory of the workspace,"
+;;;###autoload
+(defcustom lsp-scala-server-args '()
+  "Extra arguments for the Scala language server."
   :group 'lsp-scala
-  :type 'directory)
+  :type '(repeat string))
```

Comment:
I separated server-command and server-args to be more like the other modern lsp-clients.